plugin/adsk/plugin/plugin.cpp

```diff
@@ -164,6 +165,7 @@ MStatus initializePlugin(MObject obj)
     registerCommandCheck<MayaUsd::ADSKMayaUSDImportCommand>(plugin);
     registerCommandCheck<MayaUsd::EditTargetCommand>(plugin);
     registerCommandCheck<MayaUsd::LayerEditorCommand>(plugin);
+    registerCommandCheck<MayaUsd::MayaUsdInfoCommand>(plugin);
```

Comment:
Register the new `mayaUsdInfo` command which can be used (by QA, etc) to query the plugin information.